backend/crud/notification.py

```
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Session
+from models.notifications import Notification
+from schemas.notification import NotificationCreate
```

Ruff (0.8.2)

`schemas.notification.NotificationCreate` imported but unused

Remove unused import: `schemas.notification.NotificationCreate`

(F401)